apps/teams-test-app/index_cdn.html

```diff
@@ -15,8 +15,8 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <script
-      src="https://res.cdn.office.net/teams-js/2.8.0/js/MicrosoftTeams.min.js"
-      integrity="sha384-/DJ9oJEFZSpGiUQx9Na5Yb5svOPPqSb3khKxJ/YgoZ2GtrkzWgSTBpESy3LvMPVk"
+      src="https://res.cdn.office.net/teams-js/2.9.1/js/MicrosoftTeams.min.js"
```

Comment:
Why are we going from 2.8.0 to 2.9.1 instead of 2.9.0 to 2.9.1?

Reply:
Talked with Jay, he hadn't gotten around to 2.9.0 cleanup yet because he has been working on other higher priority issues this week. Makes sense, seems reasonable.